src/plugin-template.c

```diff
@@ -104,9 +132,10 @@ static bool my_plug_init(const struct clap_plugin *plugin) {
    my_plug_t *plug = plugin->plugin_data;
 
    // Fetch host's extensions here
-   plug->hostLog = plug->host->get_extension(plug->host, CLAP_EXT_LOG);
-   plug->hostThreadCheck = plug->host->get_extension(plug->host, CLAP_EXT_THREAD_CHECK);
-   plug->hostLatency = plug->host->get_extension(plug->host, CLAP_EXT_LATENCY);
+   plug->host_log = plug->host->get_extension(plug->host, CLAP_EXT_LOG);
+   plug->host_thread_check = plug->host->get_extension(plug->host, CLAP_EXT_THREAD_CHECK);
+   plug->host_latency = plug->host->get_extension(plug->host, CLAP_EXT_LATENCY);
+   plug->host_state = plug->host->get_extension(plug->host, CLAP_EXT_STATE);
```

Comment:
This template doesn't use them, but if this is meant as an idiomatic example then it's probably worthwhile to at least add a comment that you need to check whether the extension pointers are non-null before dereferencing them. Someone blindly copying this example may assume that the stored pointers are always valid (since most people never seem to check for null pointers unless explicitly told to).

Also, you may want to add explicit casts here. C allows implicitly casting between pointer types (`const void*` to `const clap_host_ext_t*`), but in C++ this is illegal and will result in a compiler error.